src/schema/v2/me/deleteUserInterestMutation.ts

```diff
@@ -19,9 +20,15 @@ export const deleteUserInterestMutation = mutationWithClientMutationId<
 >({
   name: "DeleteUserInterestMutation",
   description:
-    "Deletes a UserInterest on the logged in User's CollectorProfile.",
+    "Deletes a UserInterest on the (logged in) user or for another user.",
   inputFields: {
     id: { type: new GraphQLNonNull(GraphQLString) },
+    isOnMe: {
```

Comment:
interesting! I guess the only input to the mutation is the 'id' of an interest, so this additional snippet is needed to know which loader to use.

crazy idea...but it _looks_ like the `deleteUserInterestLoader` will correctly work in a `me` context (that is, if the user is the owner of that), and it would ofc work for an admin deleting the interest of a different user.

Maybe we can obviate the need for this field, and actually just _always_ use `deleteUserInterestLoader`?

Reply:
That does seem to work. Thanks for making the observation and pointing out this option. Changed in https://github.com/artsy/metaphysics/pull/4564/commits/68f9e45ab5df79fed28382da87446e4cd9025654

Reply:
Having switched to the loader for the delete operation, I recon we no longer need the `meDeleteUserInterestLoader`. So I am removing it then https://github.com/artsy/metaphysics/pull/4564/commits/5744dbf4fe54064c272cc341e5b4ad7293adb09c.